examples/repl.rs

```diff
@@ -198,7 +210,12 @@ mod example {
             }
             Some("kill") => println!("{:?}", context.remote()?.kill()?),
             Some("si") | Some("stepi") => println!("{:?}", context.remote()?.step()?),
-            Some("cont") | Some("continue") => println!("{:?}", context.remote()?.unpause()?),
+            Some("cont") | Some("continue") => {
+                println!("{:?}", context.remote()?.unpause()?);
+                // When we hit the next breakpoint, we also want to display the source code
+                // as does lldb and gdb.
+                print_source_for_top_of_stack_symbol(context, 3)?;
```

Comment:
This should also happen on `stepi`, `exec` and `attach` I think. Basically every command that causes execution to pause.

Reply:
I have added it to stepi. But does it make sense to add it to exec. I don't see lldb doing that.

Reply:
For exec I guess omitting it makes sense.